### ISSUE TITLE
feature request: option for not printing warnings

### ISSUE THREAD

**Issue by fblanqui** [Deducteam/Dedukti#317]
Would it be possible to add an option (say -w/--no-warnings) for not printing warnings? E.g. "[Warning] Trying to import the already loaded module ..."

**Comment by francoisthire** [comment]
Have you tried `-q`?

**Comment by fblanqui** [comment]
Right, -q doesn't print anything at all. I am not sure: perhaps it may be convenient to at least know which file is being checked. But there are other ways to know that. Feel free to close this issue if you want.
